Public/javascripts/main.js

```diff
@@ -9,12 +14,10 @@ document.addEventListener('DOMContentLoaded', function(event) {
     } while (target = target.parentElement)
   })
 
-  // There are never results when the page loads, so hide the element.
-  setElementHiddenById('results', true)
-
   // If there is a search element, configure the search callbacks.
   const queryFieldElement = document.getElementById('query')
   if (!!queryFieldElement) {
+    // When user input is entered into the query field, perform the search.
     document.addEventListener('input', function(event) {
       const searchQuery = queryFieldElement.value.trim()
       if (searchQuery.length > 0) {
```

Comment:
BTW, unrelated to this PR - I think we should debounce the search. Or is that happening and I’m not seeing it? It seems like we’re search per keystroke in my brief testing.

Reply:
Yep, debouncing was in there originally... but the fix to #128 was actually that the debounce was in slightly the wrong location! Pushing a fix to that now.